### PR TITLE
Journey Event List Container and Component

### PR DESCRIPTION
**Pull request by peterp**

### CODE REVIEW

**Comment by peterp** [comment]
@justinvdm ready for review

I will do the styling in a new pull-request.

**Comment by peterp** [comment]
@justinvdm ready for review

screen shot 2016-09-21 at 22 08 26
screen shot 2016-09-21 at 19 25 12

File: src/components/FirstEvent/__tests__/FirstEvent-tests.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  it('should render the props', () => {
+    expect(render(
+      <FirstEvent
+        firstName="John Wurst"
+        onGetStartedPress={noop}
+      />
+    )).toMatchSnapshot();
+  });
+
+  it('should be able to tap and fire `onMessagePress`', () => {
```
Comment (justinvdm) [inline src/components/FirstEvent/__tests__/FirstEvent-tests.js]: should this be `onGetStartedPress`?

File: src/containers/EventListContainer.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  );
+
+  return Object.keys(groups)
+  .sort()
+  .reverse()
+  .map(
+    date => ({
+      date,
+      label: moment(date).fromNow(),
+      events: sortBy(groups[date], event => event.occuredAt).reverse(),
```
Comment (justinvdm) [inline src/containers/EventListContainer.js]: Wondering if we should just do the `sortBy(...).reverse()` on the entire event list before doing the `groupBy`.  Would avoid needing to create a function inside a loop.

File: src/containers/FirstEventContainer.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import { connect } from 'react-redux';
+import { FirstEvent } from 'src/components';
+import { noop } from 'lodash';
+import { getAuthUserProfile } from 'src/stores/helpers';
+
+
+const mapStateToProps = state => ({
+  firstName: getAuthUserProfile(state).firstName,
+  onGetStartedPress: noop, // TODO
```
Comment (justinvdm) [inline src/containers/FirstEventContainer.js]: Should we find out what we are supposed to do when this happens?

Comment (peterp) [inline src/containers/FirstEventContainer.js]: I think the intention is to take you to the schedule call view

**Pull request merged**